modules/analysis-common/src/main/java/org/elasticsearch/analysis/common/CharGroupTokenizerFactory.java

```diff
@@ -41,6 +46,8 @@
     public CharGroupTokenizerFactory(IndexSettings indexSettings, Environment environment, String name, Settings settings) {
         super(indexSettings, settings, name);
 
+        maxTokenLength = settings.getAsInt(MAX_TOKEN_LENGTH, StandardAnalyzer.DEFAULT_MAX_TOKEN_LENGTH);
```

Comment:
Super small nitpick: can you change to use the default from CharTokenizer.DEFAULT_MAX_WORD_LEN here? Its the same value (255) as `StandardAnalyzer.DEFAULT_MAX_TOKEN_LENGTH` currently, just so we tie the default to the one thats used in the class we actually use.